tests/test_cases_actions.py

```diff
@@ -247,7 +247,7 @@ def test_handle_action_when_list_cases_calls_py42_with_correct_filters(
         connector = create_fake_connector("list_cases", mock_py42_with_case)
         connector.handle_action(param)
         mock_py42_with_case.cases.get_all.assert_called_once_with(
-            status="OPEN", assignee=_TEST_ASSIGNEE, subject=_TEST_SUBJECT
+            status="OPEN", assignee=_TEST_USER_UID, subject=_TEST_USER_UID
```

Comment:
If we wanted to get fancier here, we could change the `get_user` side effect to return different IDs based on if it received `_TESET_ASSIGNEE`, or `_TEST_SUBJECT` (and just use `_TEST_USER_UID` if neither, if that is needed)... But by doing this would be accurately testing this feature, otherwise there is a bit of gap.

Reply:
I think the main thing we care about is that `get_all()` is called with the userUid instead of the passed usernames for both of those params, so it doesn't really matter that they're different, just that the value comes from the `get_by_username()` calls.

Reply:
I suppose we could also test that `get_by_username()` was called twice since there are two params that need it called.

Reply:
Checking the call count _is_ better, but theorhetically it would still pass if we calls `_get_user()` both the assignee both times and passes the assignee's userId for the subject as well